src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/instrument-lambda-function/cloud-watch-fallback.mdx

```diff
@@ -0,0 +1,149 @@
+---
+title: "CloudWatch fallback"
+metaDescription: This doc guides you through shipping your telemetry data with Cloudwatch as a fail-safe option.
+freshnessValidatedDate: never
+---
+
+You can ship New Relic Lambda monitoring data in a few different ways depending on your needs. The option that provides the strongest performance of your function and the strongest fail-safe includes configuring CloudWatch as a fallback. This means that you will ship telemetry to New Relic through the extension and, if there's a problem with the extension, New Relic will still receive payloads through CloudWatch.
+
+Benefits of the CloudWatch fallback:
+
+* **Independent execution**: Captures all logs from CloudWatch independently of your function's execution. There's less chance it will interfere with the normal operation, or impact the invocation duration of your function.
+* **Works with secured networks**: Useful for functions in a VPC that can't have outbound traffic to New Relic.
+* **Enables enhanced monitoring**:  Needed for other forms of infra and database telemetry. To learn more, see our [Amazon RDS Enhanced Monitoring integration docmentation](/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-rds-enhanced-monitoring-integration/).
+
+While this shipping option offers the strongest fail-safe, it's important to note that the AWS CloudWatch service can generate a lot of data. Keep data ingest in mind when thinking about costs and which New Relic pricing plan you choose. 
+
+## Implementation options [#fallback]
+
+There are three ways you can implement a CloudWatch fallback:
+
+    1. **Enable the extension for payloads and disable the extension for logs**: In this scenario, payloads are shipped through the extension and logs and payloads through CloudWatch in the event the extension fails.
+    2. **Extension enabled for payloads and logs**: In this scenario, payloads and logs are shipped through the extension and payloads through CloudWatch in the event the extension fails. 
+    3. **Extension disabled**: In this scenario, CloudWatch is always used to send logs and payloads.
+
+If the Extension fails to start up, otherwise known as noop mode, or crashes, it will fail to encode the `NR_LAMBDA_MONITORING` line generated by our agents. The `NR_LAMBDA_MONITORING` line will appear in CloudWatch logs instead. If the `aws-log-ingestion` function is installed in the same region as your function, configured to ship payloads, and is triggered by your function's CloudWatch log group, then it can act as a fallback to the extension for shipping payloads.
+
+## Before you begin  [#requirements]
+
+Before you set up a fallback option you will need to complete the following:
+
+    * Install the `aws-log-ingestion` function from AWS region where you want to send your logs from.
```

Comment:
While we call it `aws-log-ingestion` internally, and the repo on GitHub has that name, the function that gets installed in the user's account is named `newrelic-log-ingestion`. That may be confusing initially, but it does make sense, depending on the context (within NR, vs within AWS).

Reply:
Thanks @mrickard ! I have renamed this in the doc.